website/content/en/v0.27.0/getting-started/getting-started-with-eksctl/_index.md

```diff
@@ -123,7 +123,7 @@ Karpenter should terminate the now empty nodes.
 
 This section describes optional ways to configure Karpenter to enhance its capabilities.
 In particular, the following commands deploy a Prometheus and Grafana stack that is suitable for this guide but does not include persistent storage or other configurations that would be necessary for monitoring a production deployment of Karpenter.
-This deployment includes two Karpenter dashboards that are automatically onboarded to Grafana.They provide a variety of visualization examples on Karpenter metrics.
+This deployment includes two Karpenter dashboards that are automatically onboarded to Grafana. They provide a variety of visualization examples on Karpenter metrics.
```

Comment:
Thanks for this change! Can you propagate this change throughout all versions where it has occurred?

Reply:
Updated.